source/common/router/router.cc

```diff
@@ -2118,7 +2118,7 @@ void Filter::maybeProcessOrcaLoadReport(const Envoy::Http::HeaderMap& headers_or
   auto host = upstream_request.upstreamHost();
   const bool need_to_send_load_report =
       (host != nullptr) && cluster_->lrsReportMetricNames().has_value();
-  if (!need_to_send_load_report) {
+  if (!need_to_send_load_report && !orca_load_report_cb_) {
```

Comment:
nit: please update the comment above.

Reply:
Hrm, I *think* it is already correct as it says "or invoke the ORCA callbacks.", what am I missing?